ios/XMTPModule.swift

```diff
@@ -582,6 +578,17 @@ public class XMTPModule: Module {
 			return results
 		}
 
+		AsyncFunction("getHmacKeys") { (installationId: String) -> [UInt8] in
+			guard
+				let client = await clientsManager.getClient(key: installationId)
+			else {
+				throw Error.noClient
+			}
+			let hmacKeys = try await client.conversations.getHmacKeys()
```

Comment:
No need for async here

Reply:
So getHmacKeys is not async but because conversations is an actor for thread safety it has to be be awaited.